embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/payload/Session.kt

```diff
@@ -132,9 +134,9 @@ internal data class Session @JvmOverloads internal constructor(
      * Enum to discriminate the different ways a session can start / end
      */
     enum class SessionLifeEventType {
-        @SerializedName("s")
-        STATE, @SerializedName("m")
-        MANUAL, @SerializedName("t")
+        @Json(name = "s")
+        STATE, @Json(name = "m")
```

Comment:
Nit: I know it's already there, but do you mind fixing the wrapping of the annotation so it appears in a separate line?

Reply:
Fixed